virtcontainers/utils/utils_linux_test.go

```diff
@@ -36,16 +36,18 @@ func TestFindContextID(t *testing.T) {
 
 func TestGetDevicePathAndFsTypeEmptyMount(t *testing.T) {
 	assert := assert.New(t)
-	_, _, err := GetDevicePathAndFsType("")
+	_, _, _, err := GetDevicePathAndFsTypeOptions("")
 	assert.Error(err)
 }
 
 func TestGetDevicePathAndFsTypeSuccessful(t *testing.T) {
 	assert := assert.New(t)
+	opts := []string{"rw", "nosuid", "nodev", "noexec", "relatime"}
 
-	path, fstype, err := GetDevicePathAndFsType("/proc")
+	path, fstype, fsOptions, err := GetDevicePathAndFsTypeOptions("/proc")
 	assert.NoError(err)
 
 	assert.Equal(path, "proc")
 	assert.Equal(fstype, "proc")
+	assert.True(DeepCompare(fsOptions, opts))
```

Comment:
Can we actually rely on the options for /proc being the same on every distro?

Reply:
> Can we actually rely on the options for /proc being the same on every distro?

On Fedora and Ubuntu I didn't find any difference. But you are right. Any suggestions on how to handle this for the specific unit test ?

Reply:
I think @dgibson is right - it's going to be safer (and more future-proof) to make `TestGetDevicePathAndFsTypeSuccessful()` effectivley do `grep /proc /proc/mounts` and compare it to the output of `GetDevicePathAndFsTypeOptions("/proc")`.